package.json

```diff
@@ -4,24 +4,35 @@
   "description": "A simple style guide generator for Grunt.js",
   "main": "Gruntfile.js",
   "dependencies": {
-    "nunjucks": "3.1.6",
+    "nunjucks": "3.2.0",
     "underscore": "1.8.3",
-    "yaml-front-matter": "3.2.3"
+    "yaml-front-matter": "4.1.0"
   },
   "devDependencies": {
-    "grunt": "1.0.3",
+    "@commitlint/cli": "8.3.4",
+    "@silvermine/eslint-config": "2.3.0",
+    "check-node-version": "4.0.2",
+    "grunt": "1.0.4",
     "grunt-browserify": "5.3.0",
-    "grunt-contrib-jshint": "2.0.0",
-    "grunt-sass": "3.0.2"
+    "grunt-contrib-jshint": "2.1.0",
+    "grunt-sass": "3.1.0",
+    "node-sass": "4.13.0"
   },
   "peerDependencies": {
-    "grunt": "^0.4.5",
-    "grunt-browserify": "^3.8.0",
-    "grunt-sass": "^2.0.0"
+    "grunt": "~1.0.0",
```

Comment:
Why did these change from carets to tildes? It seems like we should keep the less restrictive `^` so that this grunt plugin remains usable with a wider range of versions that people may use.